skyportal/models.py

```diff
@@ -159,6 +161,17 @@ class Obj(Base):
 
     followup_requests = relationship('FollowupRequest', back_populates='obj')
 
+    @hybrid_property
+    def last_detected(self):
```

Comment:
More of a note to myself: currently, K relies on `last_detected` to figure out which prv_candidates to post to SP if an alert passes a filter and a candidate with the objectId already exists in SP. Follow-up photometry posted to the candidate will obviously break this. I'm making K track which objectId's passed which filters when (for the last time) and so try to post stuff that came later.

Reply:
Yeah thats a good plan. It is correct that Non-alert photometry will break this